### ISSUE TITLE
for batch size more than one

### ISSUE THREAD

**Issue by mohammadshahabuddin** [jacobgil/pytorch-grad-cam#307]
Could you please let me know how should I mention targets when batch size is more than one? For batch size=1, the code below is working. But, for batch size more than one how should i mention targets? 

        for i, (images, labels) in enumerate(train_loader2):
            # Run the forward pass
            images = images.cuda()
            labels = labels.cuda()

            outputs = model(images)
            loss1 = criterion(outputs, labels)

            class_loss = torch.mean(loss1)

            targets = [ClassifierOutputTarget(labels.item())]

            grayscale_cam = cam(input_tensor=images, targets=targets)

**Comment by jacobgil** [comment]
Hi,
Just a list with the target for every member in the batch:

```python
targets = [ClassifierOutputTarget(l.item()) for l in labels]
```

**Comment by mohammadshahabuddin** [comment]
Hi,

Great! Thank you so much!
